api/article/views.py

```diff
@@ -7,8 +9,23 @@ class ArticleViewSet(viewsets.ModelViewSet):
     serializer_class = ArticleSerializer
     permission_classes = permissions.IsAuthenticatedOrReadOnly,
 
+    serializer_class_map = {
+        'retrieve': ArticleSerializer,
+        'list': ArticleSerializer,
+        'create': ArticleSerializer,
+        'self_articles': ArticleSerializer,
```

Comment:
Name it just `atricles`, i think it will be enough.

Reply:
This endpoint is already taken. I'll think about changing the name in the future.